src/subscription_manager/repofile.py

```diff
@@ -65,6 +65,7 @@ class Repo(dict):
             'sslclientkey': (0, None),
             'sslclientcert': (0, None),
             'metadata_expire': (1, None),
+            'arches': (1, '0'),
```

Comment:
I would make 'arches' debian specific property, because it is added to redhat.repo and it is absolutely useless there.

Reply:
can you give me a hint what needs to be adapted?